Configuration/PyReleaseValidation/test/BuildFile.xml

```diff
@@ -2,3 +2,29 @@
 </test>
 <test name="test-runTheMatrix-interactive" command="test-runTheMatrix_interactive.sh">
 </test>
+
+<!-- Tests for MC production for 2022 and 2023 -->
+<!-- 
+    The conditions used here for 2022 and 2023 will be changed 
+    to use the proper GT from ALCA including the BS smearing 
+    record from GT. For the moment we use the automatic ones.
+    By default these tests won't fail if a failure happens at
+    HLT step. If you want to catch those errors please set the
+    environment variable CMSSW_MC_SETUP_TEST_CATCH_HLT.
+-->
+
+<!-- In CMSSW_13_0_18 the auto:phase1_2023_realistic (pre BPix) is 130X_mcRun3_2023_realistic_postBPix_v1 -->
+<test name="test_MC_23_setup" command="test_mc_setup/test_MC_setup.sh 140X_mcRun3_2023_realistic_v3 Run3_2023 2023v12 CMSSW_13_0_18 130X_mcRun3_2023_realistic_v13 Realistic25ns13p6TeVEarly2023Collision" />
```

Comment:
why the SimBeamSpot cannot be picked up from GT? (option `DBRealistic`)

Reply:
With `140X` it could. But I think that is the mistake here.

We would like to run the production with the original conditions used for the `Summer[22|23]` campaigns (so `124X` and `130X` GTs). And those GTs has no `SimBeamSpotObjectsRcd` included. Now the plan with @cms-sw/alca-l2 would be to have uniform and compatible : 
- `140X` + `130X` GTs for 2023;
- `140X` + `124X`   GTs for 2022.
To have exactly the same conditions + BS picked from GT. 

For the moment to be uniform I would use the `130X` GT for 2023 and `124X` GT for 2022.